terraform/eks/e2e/main.tf

```diff
@@ -34,9 +34,9 @@ resource "aws_eks_node_group" "this" {
   subnet_ids      = module.basic_components.public_subnet_ids
 
   scaling_config {
-    desired_size = 1
-    max_size     = 1
-    min_size     = 1
+    desired_size = var.nodes
+    max_size     = var.nodes
+    min_size     = var.nodes
```

Comment:
This is not blocking but shouldn't we be able to configure max and min size separately?

Reply:
We can discuss this, but right now I don't see a use-case for doing anything beyond the desired size.